packages/3dt/buildinfo.json

```diff
@@ -4,8 +4,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/3dt.git",
-    "ref": "bf8b5584843fa332bafe23d7e3801b137baf2dd8",
-    "ref_origin": "master"
+    "ref": "ee0963026a71f88f836b832e51c3575798b7396f",
+    "ref_origin": "mnaboka/3dt-fix"
```

Comment:
@orsenthil yes. Work In Progress label means "don't review this yet" / "it contains non-production code, missing tests, etc". Things get pointed to arbitrary dev locations, and that's actually a good thing(tm)